src/components/Login.tsx

```diff
@@ -2,11 +2,11 @@ import styled from 'styled-components/macro'
 import heroImg from '../assets/images/Login-hero.jpg'
 import Button from './common/Button/Button'
 
-const Login = (): JSX.Element => {
-  const authEndpoint: string = 'https://accounts.spotify.com/authorize'
-  const redirectUri: string = process.env.REACT_APP_REDIRECT_URI as string
-  const clientId: string = process.env.REACT_APP_CLIENT_ID as string
-  const scope: string = 'user-read-private'
+function Login(): JSX.Element {
+  const authEndpoint = 'https://accounts.spotify.com/authorize'
+  const redirectUri = process.env.REACT_APP_REDIRECT_URI as string
```

Comment:
If you like, i can show you how to set the api call into the serverless api folder in vercel, so you call here just your local route